pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/JavaInstanceMain.java

```diff
@@ -172,6 +175,7 @@ public void start() throws Exception {
         }
         sourceDetailsBuilder.setSubscriptionType(Function.SubscriptionType.valueOf(sourceSubscriptionType));
         sourceDetailsBuilder.putAllTopicsToSerDeClassName(new Gson().fromJson(sourceTopicsSerdeClassName, Map.class));
+        sourceDetailsBuilder.setTopicsPattern(topicsPattern);
```

Comment:
since topicPattern is not set as a required field, this can be null.  Can we do a null check before setting?

Reply:
good point.. let me fix it.